### PR TITLE
Upgrade event bus dependencies to kubernetes 1.12.6

### PR DESCRIPTION
**Pull request by montaro**
See also #5168

### CODE REVIEW

File: components/event-bus/cmd/event-publish-service/handlers/publish.go [hunk 1]
```diff
@@ -3,7 +3,7 @@ package handlers
 import (
 	"encoding/json"
 
-	"github.com/kyma-project/kyma/components/event-bus/api/publish/v1"
+	v1 "github.com/kyma-project/kyma/components/event-bus/api/publish/v1"
```
Comment (anishj0shi) [inline components/event-bus/cmd/event-publish-service/handlers/publish.go]: i see there's a reference name "publishv2" can we be consistent here ?

File: components/event-bus/internal/knative/util/kn-eventing_test.go [hunk 2]
```diff
@@ -163,7 +167,8 @@ func Test_SendMessage(t *testing.T) {
 	assert.Nil(t, err)
 	u, err := url.Parse(srv.URL)
 	assert.Nil(t, err)
-	ch.Status.SetAddress(u.Hostname() + ":" + u.Port())
+	url := &apis.URL{Scheme: "http", Host: fmt.Sprintf("%s:%s", u.Hostname(), u.Port())}
+	ch.Status.SetAddress(url)
```
Comment (anishj0shi) [inline components/event-bus/internal/knative/util/kn-eventing_test.go]: for readability, can we create "url" as a value and pass it's reference in SetAddress()? something like SetAddress(&url)

Comment (montaro) [reply components/event-bus/internal/knative/util/kn-eventing_test.go]: Thanks for the comment.
IMHO, it's readable and I'd not also change the signature of the `SetAddress()`

File: resources/event-bus/charts/event-publish-service/templates/deployment.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
```
Comment (sayanh) [inline resources/event-bus/charts/event-publish-service/templates/deployment.yaml]: May be, we can move these changes to another PR(will be created anyway) which will have the bumped image of event-bus.
